scripts/prod-build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -e
+
+$REGISTRY="prd-mama-router.ng.prk-host.net:5000/"
```

Comment:
Surely you can just do `export REGISTRY="prd-mama-router.ng.prk-host.net:5000/"`?